### PR TITLE
Add listing for SmartBatteries and related tests

### PR DESCRIPTION
**Pull request by berendhaan**
This PR adds a new method to the API to list the current users connected batteries, for example the Sessy.

If you see something let me know :-) I'm not a python dev by day (or night)

In a next PR I will add a method to get the 'earnings' per connected Smart Battery of the user.

### CODE REVIEW

**Comment by DCSBL** [comment]
Oh awesome. Will take a look at this asap!

**Comment by berendhaan** [comment]
The failing flake8 task does not look (completely) related to this PR

`unable to find qualified name for module: example.py
python_frank_energie/models.py:426:1: D400 First line should end with a period
python_frank_energie/models.py:450:1: D106 Missing docstring in public nested class
tests/test_smart_batteries.py:1:1: D100 Missing docstring in public module`

File: tests/test_smart_batteries.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import aiohttp
```
Comment (DCSBL) [inline tests/test_smart_batteries.py:1]:
```suggestion
"""Tests for Frank Energie Smart batteries implementation."""
import aiohttp
```

Comment (DCSBL) [inline tests/test_smart_batteries.py:1]:
```suggestion
"""Tests for Frank Energie Smart batteries implementation.""" 
import aiohttp
```

File: python_frank_energie/models.py [hunk 2]
```diff
@@ -419,3 +419,59 @@ def from_userprices_dict(data: dict[str, str]) -> MarketPrices:
...
+        id: str
+        max_charge_power: float
+        max_discharge_power: float
+        provider: str
+        created_at: datetime
+        updated_at: datetime
+
+        @staticmethod
+        def from_dict(payload: dict[str, str]) -> SmartBatteries.SmartBattery:
+            """Parse the response from the me query."""
```
Comment (DCSBL) [inline python_frank_energie/models.py:464]:
```suggestion
            """Parse the response from the SmartBatteries query."""
```

File: python_frank_energie/models.py [hunk 3]
```diff
@@ -419,3 +419,59 @@ def from_userprices_dict(data: dict[str, str]) -> MarketPrices:
             electricity=PriceData(customerMarketPrices.get("electricityPrices")),
             gas=PriceData(customerMarketPrices.get("gasPrices")),
         )
+
+
+@dataclass
+class SmartBatteries:
+    """Collection of the users SmartBatteries"""
```
Comment (DCSBL) [inline python_frank_energie/models.py:426]:
```suggestion
    """Collection of the users SmartBatteries."""
```

File: python_frank_energie/models.py [hunk 4]
```diff
@@ -419,3 +419,59 @@ def from_userprices_dict(data: dict[str, str]) -> MarketPrices:
...
+
+        return SmartBatteries(
+            smart_batteries=[
+                SmartBatteries.SmartBattery.from_dict(smart_battery)
+                for smart_battery in payload.get("smartBatteries")
+            ],
+        )
+
+    @dataclass
+    class SmartBattery:
```
Comment (DCSBL) [inline python_frank_energie/models.py:450]: """Collection of the users SmartBatteries."""
```suggestion
    class SmartBattery:
        """SmartBattery model."""
```

File: tests/test_smart_batteries.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
+"""Tests for Frank Energie Smart batteries implementation."""
```
Comment (DCSBL) [inline tests/test_smart_batteries.py:1]:
```suggestion
"""Tests for Frank Energie Smart batteries implementation."""
```

**Review by DCSBL — APPROVED** [review]
I have now way to test this, but it looks good! The added unit test is nice!

Thanks!

**Pull request merged**